helm/fiftyone-teams-app/README.md.gotmpl

```diff
@@ -95,12 +95,13 @@ documentation before completing your upgrade.
 
 Voxel51 recommends upgrading your deployment using
 [`legacy` authentication mode](https://docs.voxel51.com/teams/pluggable_auth.html#legacy-mode)
-and
-[migrating](https://docs.voxel51.com/teams/pluggable_auth.html#migrating-from-legacy-to-internal-mode)
-to
```

Comment:
In a previous PR, we updated README.md not the template. So when my pre-commit hooks ran, these changes got overrwritten.  Fixed it by adding to the template.

Reply:
I still think changes in this directory are going to trigger the release Actions and you'll wind up with a new helm chart?  Maybe?  You know more about it than I do at this point.

Reply:
It might? I haven't deep dove into the tooling to understand it's behavior.  I'll just remove this from the PR.

Reply:
You could also just target these changes at a 1.7.0 release branch...

Reply:
If there was one, I would. I guess I'll just create a release/v1.7.0 branch.